app/deliver_grant_funding/templates/deliver_grant_funding/grant_team/grant_user_list.html

```diff
@@ -0,0 +1,111 @@
+{% extends "deliver_grant_funding/base.html" %}
+{% from "govuk_frontend_jinja/components/table/macro.html" import govukTable %}
+{% from "govuk_frontend_jinja/components/notification-banner/macro.html" import govukNotificationBanner %}
+{% from "govuk_frontend_jinja/components/button/macro.html" import govukButton %}
+
+{% block pageTitle %}
+  Grant team - {{ grant.name }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_item_identifier = "grant_team" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% with messages = get_flashed_messages() %}
+        {% if messages %}
+          {% set user_add_success_message %}
+            <h3 class="govuk-notification-banner__heading">Grant team member added</h3>
+            <p class="govuk-body">{{ messages[0] }}</p>
+          {% endset %}
+          {{
+            govukNotificationBanner({
+              "html": user_add_success_message, "type": "success"
+            })
+          }}
+        {% endif %}
+      {% endwith %}
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl">{{ grant.name }}</span>
+        Grant team
+      </h1>
+      {% if current_user.is_platform_admin %}
+        {{
+          govukButton({
+             "text": "Add grant team member",
+             "href": url_for("deliver_grant_funding.add_user_to_grant", grant_id=grant.id)
+          })
+        }}
+      {% endif %}
+    </div>
+  </div>
+
+  {% set table_pending_signin_rows = namespace(items=[]) %}
+  {% if users %}
+    {% for user in users if user.name|lower == '' %}
```

Comment:
This `user.name` attribute doesn't exist. I guess this is because you've broken the PR into frontend and backend, so the `name` field will come later.

Feedback for the future, but splitting the work this way doesn't feel right. Either the backend should come first, or we should do them as fullstack PRs. It's a smaller PR, but this hasn't made it easier to review.

Reply:
No, I don't get errors.